core/producers/trigAprProducers.fcl

```diff
@@ -44,6 +44,10 @@ TrigAprProducers : {
     FitDirection     : 0
   }
 
+  TTAprKSFMC: {
```

Comment:
I thought these were being defined in `Mu2e/Production` rather than in `Mu2e/mu2e_trig_config`?

Reply:
Looks like I missed an instance when cleaning up: I'll fix this.